easybuild/scripts/install_eb_dep.sh

```diff
@@ -1,77 +1,81 @@
 #!/bin/bash
 
-set -e
-
 if [ $# -ne 2 ]; then
     echo "Usage: $0 <name>-<version> <prefix>"
     exit 1
 fi
-PKG=$1
-PREFIX=$2
 
-PKG_NAME=`echo $PKG | sed 's/-[^-]*$//g'`
-PKG_VERSION=`echo $PKG | sed 's/.*-//g'`
+set -eu
+
+PKG="$1"
+PREFIX="$2"
+
+PKG_NAME="${PKG%-*}"
+PKG_VERSION="${PKG##*-}"
 
 CONFIG_OPTIONS=
 PRECONFIG_CMD=
 
-if [ x$PKG_NAME == 'xmodules' ] && [ x$PKG_VERSION == 'x3.2.10' ]; then
+if [ "$PKG_NAME" == 'modules' ] && [ "$PKG_VERSION" == '3.2.10' ]; then
```

Comment:
Use = and not == for POSIX comformance in all if [ statements comparing strings

Reply:
The script has a bash shebang and AFAIK == is bash and worked before. Don't really want to change something like that unless you see value in doing so